From 29f0de401b9df056c7eeb47a025c341654ca579d Mon Sep 17 00:00:00 2001
From: Andrej Valek <andrej.valek@siemens.com>
Date: Thu, 14 Jun 2018 08:54:14 +0200
Subject: [PATCH] secret-storage: rename PAGESIZE variables to pagesize

PAGESIZE is a define in some libc libraries, which conflicts with these
local variables.

Upstream-Status: Submitted [https://github.com/balabit/syslog-ng/pull/2111]

Signed-off-by: Andrej Valek <andrej.valek@siemens.com>
Signed-off-by: Marko Peter <peter.marko@siemens.com>
---
 lib/secret-storage/nondumpable-allocator.c            | 4 ++--
 lib/secret-storage/tests/test_nondumpable_allocator.c | 6 +++---
 lib/secret-storage/tests/test_secret_storage.c        | 4 ++--
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/lib/secret-storage/nondumpable-allocator.c b/lib/secret-storage/nondumpable-allocator.c
index 07d94fbb0..314ce571f 100644
--- a/lib/secret-storage/nondumpable-allocator.c
+++ b/lib/secret-storage/nondumpable-allocator.c
@@ -104,8 +104,8 @@ gpointer
 nondumpable_buffer_alloc(gsize len)
 {
   gsize minimum_size = len + ALLOCATION_HEADER_SIZE;
-  gsize PAGESIZE = sysconf(_SC_PAGE_SIZE);
-  gsize alloc_size = round_to_nearest(minimum_size, PAGESIZE);
+  gsize pagesize = sysconf(_SC_PAGE_SIZE);
+  gsize alloc_size = round_to_nearest(minimum_size, pagesize);
 
   Allocation *buffer = _mmap(alloc_size);
   if (!buffer)
diff --git a/lib/secret-storage/tests/test_nondumpable_allocator.c b/lib/secret-storage/tests/test_nondumpable_allocator.c
index 6ef5b3035..a20b287ff 100644
--- a/lib/secret-storage/tests/test_nondumpable_allocator.c
+++ b/lib/secret-storage/tests/test_nondumpable_allocator.c
@@ -34,10 +34,10 @@ Test(nondumpableallocator, malloc_realloc_free)
   strcpy(buffer, test_string);
   cr_assert_str_eq(buffer, test_string);
 
-  const gsize PAGESIZE = sysconf(_SC_PAGE_SIZE);
-  gpointer buffer_realloc = nondumpable_buffer_realloc(buffer, 2*PAGESIZE);
+  const gsize pagesize = sysconf(_SC_PAGE_SIZE);
+  gpointer buffer_realloc = nondumpable_buffer_realloc(buffer, 2*pagesize);
   cr_assert_str_eq(buffer_realloc, test_string);
-  ((gchar *)buffer_realloc)[2*PAGESIZE] = 'a';
+  ((gchar *)buffer_realloc)[2*pagesize] = 'a';
 
   nondumpable_buffer_free(buffer_realloc);
 }
diff --git a/lib/secret-storage/tests/test_secret_storage.c b/lib/secret-storage/tests/test_secret_storage.c
index 5297d4f60..89122cada 100644
--- a/lib/secret-storage/tests/test_secret_storage.c
+++ b/lib/secret-storage/tests/test_secret_storage.c
@@ -263,11 +263,11 @@ Test(secretstorage, test_rlimit)
   cr_assert(!getrlimit(RLIMIT_MEMLOCK, &locked_limit));
   locked_limit.rlim_cur = MIN(locked_limit.rlim_max, 64 * 1024);
   cr_assert(!setrlimit(RLIMIT_MEMLOCK, &locked_limit));
-  const gsize PAGESIZE = sysconf(_SC_PAGE_SIZE);
+  const gsize pagesize = sysconf(_SC_PAGE_SIZE);
 
   gchar *key_fmt = g_strdup("keyXXX");
   int i = 0;
-  int for_limit = locked_limit.rlim_cur/PAGESIZE;
+  int for_limit = locked_limit.rlim_cur/pagesize;
   for (; i < for_limit; i++)
     {
       sprintf(key_fmt, "key%03d", i);
-- 
2.11.0

